pkgs/games/factorio/default.nix

```diff
@@ -10,7 +10,7 @@ assert releaseType == "alpha" || releaseType == "headless" || releaseType == "de
 
 with stdenv.lib;
 let
-  version = if releaseType != "demo" then "0.15.16" else "0.14.23";
+  version = if releaseType != "demo" then "0.15.26" else "0.15.25";
```

Comment:
@Mic92 Thanks, I can confirm that now. Pretty cool game, judging by their media.